Deck_Shuffler/Deck_Shuffler/Shuffler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    class Shuffler
+    {
+        public static Deck Shuffle(Deck DeckToShuffle)
+        {
+            Random rnd = new Random();
+            List<int> indexes = Enumerable.Range(0, 52).ToList();
```

Comment:
Do you understand the implications of calling ToList on an IEnumerable? Do you understand what the difference is between an IEnumerable and and IList? In this instance, I think it's fine because of the call to indexes.Count, which is cheap compared to e.g. indexes.Any() or indexes.Count(). But it might be worth understanding it if you don't already as this sort of stuff crops up a lot in interviews.